hooks/src/index.js

```diff
@@ -21,6 +21,12 @@ options.render = vnode => {
 };
 
 
+let oldDiffed = options.diffed;
+options.diffed = () => {
+	if (oldDiffed) oldDiffed();
+	currentComponent = undefined;
+};
+
```

Comment:
There is another `options.diffed` hook just below. Any reason not to fold the `currentComponent` assignment into that?

Reply:
Thought it would improve readability a bit, I'll move it